src/controllers/words.js

```diff
@@ -64,18 +64,39 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
-    if (!words.length) {
-      query = searchEnglishRegexQuery(regexKeyword);
-      const englishWords = await searchWordUsingEnglish({ query, ...searchQueries });
-      return packageResponse({
-        res,
-        docs: englishWords,
-        model: Word,
-        query,
-        ...rest,
-      });
+    let words;
+    let query;
+    if (searchWord.match(/".*"/) || searchWord.match(/'.*'/)) {
+      const regexKeywordWithoutQuotes = new RegExp(
```

Comment:
This could be simplified to:

```js
const regexKeywordWithoutQuotes = new RegExp(regexKeyword.toString().replaceAll(/["']/, ''), 'i');
```

This will convert `regexKeyword` into a string, which will allow us to call the `replaceAll` function which will target all the double and single quotes and replace them with empty strings. We then attach the `i` flag for case insensititivity.